AGENTS.md

```
@@ -0,0 +1,250 @@
+| `sections` | ESN sections, section spaces, memberships |
+| `universities` | University database |
+| `utils` | Shared utilities |
+| `wiki` | Wiki fetcher integration |
+
+---
+
+## Settings Architecture
+
+Settings use **`django-configurations`** — class-based with mixin composition. Activated via the `DJANGO_CONFIGURATION` env var.
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)